ios/fastlane/lanes/lane_tests.rb

```diff
@@ -0,0 +1,15 @@
+desc "Tests given scheme"
+desc "Parameters:"
+desc "- 'scheme : <value>' to define scheme to test"
+desc " "
+desc "Example usage: fastlane test_build scheme:'NativeSigner'"
+lane :test_build do |options|
+  scheme = options[:scheme]
+  clear_derived_data
+  scan(
+    clean: true,
+    scheme: scheme,
+    device: "iPhone 13",
```

Comment:
Would it make sense to run tests on the latest iPod touch? I know it's not supported any more, but that's what people have

Reply:
iPod Touch gen 7 will be removed from available simulators in September for newer XCode versions, given iOS 16 release.
As we don't have device-specific unit tests for now and I don't think we'll have for foreseeable future, only requirement here is to use device that is accessible in all versions of XCode 🙏🏻